### ISSUE TITLE
[RTL433] issue to retrieve data

### ISSUE THREAD

**Issue by jlighty8** [domoticz/domoticz#4124]
Hello,
Currently I cannot retrieve the metrics from my probes using rtl433.

Domoticz version : 2020.2 (build 12004)

I can see these error messages inside the application log

2020-05-01 16:55:12.654 Status: Rtl433: Worker started...
2020-05-01 16:55:12.859 Status: Rtl433: Unhandled sensor reading, please report: (time,model,id,channel,battery,temperature_C,mic,rid,button,humidity,state,brand,rain_rate,rain_total,gust,average,direction,pressure_hPa,uv,power_W,energy_kWh,unit,group_call,command,dim,dim_value,wind_speed,wind_gust,wind_direction,dipswitch,rbutton,device,temperature_F,rc,brandmodelidtemperature_C,setpoint_C,switch,direction_str,direction_deg,speed,rain,msg_type,signal,hours,minutes,seconds,year,month,day,sensor_code,uv_status,uv_index,lux,wm,fc,ws_id,rainfall_mm,wind_speed_ms,gust_speed_ms,current,interval,learn,windstrength,winddirection,flags,maybetemp,binding_countdown,depth,power0,power1,power2,node,ct1,ct2,ct3,ct4,Vrms/batt,temp1_C,temp2_C,temp3_C|,temp4_C,temp5_C,temp6_C,pulse,sid,transmit,moisture,status,type,pressure_PSI,battery_mV,pressure_bar,code,pulses,energy,device id,len,to,from,payload,event,heartbeat,brandmodelidstatus,temperature_C1,temperature_C2,test,newbattery,heating,heating_temp,water,counter,alarm,depth_cm,repeat,temperature_1_C,temperature_2_C 
then many Failure errors like this following one
2020-05-01 16:58:14.523 Status: Rtl433: Failure! Retrying in 30 seconds...

Running manually the command rtl_433 works and it detects my probes
here is the output :
Registering protocol [1] "Rubicson Temperature Sensor"
................
Registering protocol [77] "Thermopro TP12 Thermometer"
Registering protocol [78] "GE Color Effects"
Registered 78 out of 98 device decoding protocols
Found 1 device(s)

trying device  0:  Realtek, RTL2838UHIDIR, SN: 00000001
Detached kernel driver
Found Rafael Micro R820T tuner
Using device 0: Generic RTL2832U OEM
Exact sample rate is: 250000.000414 Hz
[R82XX] PLL not locked!
Sample rate set to 250000.
Bit detection level set to 0 (Auto).
Tuner gain set to Auto.
Reading samples in async mode...
Tuned to 433920000 Hz.
Allocating 15 zero-copy buffers
2020-05-01 17:09:34 	:	OS :	THN132N
	House Code:	 148
	Channel:	 1
	Battery:	 OK
	Celcius:	 12.90 C

I remembered this device worked on Domoticz 2 months ago.

Thank for your help

Regards

**Comment by gizmocuz** [comment]
@jlighty8 , did you do a search here for other issues related to RTL433 ? There have been some discussions and improvements, could as well be your RTL433 version or a commandline parameter.
Best to use the search option

**Comment by jlighty8** [comment]
@gizmocuz  of course before creating this issue, I searched all topics related to RTL433 created since February and I found nothing relevant.
I read the Domoticz source code to see how the rtl433 command is called from Domoticz.
It seems from the 10th February, the rtl433 call changed. It uses a new parameters  "-M newmodel -C si ".
My rtl433 version is 0.1-366-gaf79197 (raspbian buster) and it doesn't support these options.
I'm going to check another version maybe I have to compile it.
